### ISSUE TITLE
Make installation instructions stable

### ISSUE THREAD

**Issue by michaelwoerister** [rust-lang/measureme#182]
https://github.com/rust-lang/measureme/pull/177 changed installation instructions to a specific tag which is much better than making them install whatever is on the main branch at the moment. However, it would be even better if the installation string could stay immutable regardless of what the current version is.

Now that we support old file formats too, we could introduce a "stable" branch that always points to the latest release. That would enable us to make the installation string something like:

```bash
cargo install --git https://github.com/rust-lang/measureme --branch stable crox
``` 

We could then also update the instructions in https://blog.rust-lang.org/inside-rust/2020/02/25/intro-rustc-self-profile.html, which is still a common entry point to self-profiling, I think (at least I always point folks to that).

**Comment by rylev** [comment]
I'd be in favor of this change. However, what speaks against just releasing all the crates in the tool suite on crates.io so that users can install with `cargo install`?

**Comment by michaelwoerister** [comment]
Right, that would be an option too. The only thing that speaks against that is that the tool names are a bit too generic right now, e.g. `flamegraph`, `stack_collapse`, `summarize`, ... Is there a way to "namespace" the tools?

**Comment by rylev** [comment]
Unfortunately not - we'd have to change their names which is unfortunate... Perhaps the right long term solution is to create the metabinary, and then release that on crates.io

**Comment by martin-t** [comment]
The name `flamegraph` conflicts with an already existing tool - https://crates.io/crates/flamegraph. I find this confusing - i had the other tool installed (it seems installing one silently overwrites the other) and took me a while to realize what was happening.
